.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: php
 
+sudo: false
+
+cache:
+    directories:
+      - $HOME/.composer/cache
```

Comment:
I suggest caching only the ``/files`` subfolder. this will avoid to invalidate the Travis cache (forcing to persist the updated cache) because of the composer repo cache which will be updated on each build (because Packagist will be updated between all builds)